paper/paper.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'MLJFlux: Deep learning interface to the MLJ toolbox'
+tags:
+  - Julia
+  - Machine learning
+  - Deep learning
+authors:
+  - name: Ayush Shridhar
+    orcid: 0000-0003-3550-691X
+    affiliation: 1
+  - name: Anthony Blaom
+    affiliation: 2
+affiliations:
+ - name: International Institute of Information Technology, Bhubaneswar, India
+   index: 1
+ - name: Department of Computer Science, University of Auckland
+   index: 2
+date: 26 March 2021
+bibliography: paper.bib
+---
+
+# Introduction
+
+We present _MLJFlux.jl_ [@MLJFlux.jl], an interface between the _MLJ_ [@Blaom2020] machine learning toolbox and _Flux.jl_ [@Innes2018] deep learning framework written in the _Julia_ [@Julia-2017] programming language. MLJFlux makes it possible to implement supervised deep learning models while adhering to the MLJ workflow. This means that users familiar with the MLJ design can write their models in Flux with a few slight modifications and perform all tasks provided by the MLJ model spec which includes evaluation, fitting on a dataset, hyperparameter tuning or plotting. The interface also provides options to train the model on different hardware and warm start the model after changing some specific hyper-parameters.
```

Comment:
The citations should not interrupt the flow of the sentence. So, for instance, 

"... between the MLJ [@Julia-2017] machine learning toolbox and ..." ✔️ 

"... between the MLJ machine learning toolbox [@Julia-2017] and ..."  ❌ 

This applies to the other 2 citations in this paragraph